api/main.py

```diff
@@ -43,6 +43,10 @@ def _check_float(value: Union[float,None], field_name: str) -> Union[float,None]
             value = value / 100
     return value
 
+@app.get("/status")
+def get_status():
+    return {"status": "ok"}
+
```

Comment:
- make sure this is not there in the docs
- strong typing response - status: ok can be it's own object